policies/adding-new-maintainers.md

```diff
@@ -0,0 +1,23 @@
+This document describes the process for adding new people to maintainer roles.
+"Maintainer" here basically means any role with github write powers. The
```

Comment:
```suggestion
Maintainers are the people who keep a package or sub-package working. In particular, they review and merge PRs, keep track of what's going on and organize ideas for future development (see https://www.astropy.org/team.html#Subpackage_maintainer for details). Therefore, a maintainer usually has experience with a package e.g. through work they did themselves or by commenting and helping others to work on PRs in a package or sub-package. There is no formal minimum on the number of commits/PRs/etc. though, anyone can volunteer or be suggested as a maintainer following the process below.

This document describes the process for adding new people to maintainer roles.
"Maintainer" here basically means any role with github write powers. The
```

Reply:
We don't have the mentoring program yet, but this would be a good place to give a shout out about that.

Reply:
(I'm accepting this suggestion, but am going to modify it a bit further in a commit to push up)